src/docs/project-maintainership.md

```diff
@@ -158,9 +124,17 @@ While we will archive by request of maintainers of any Wayfair project, we would
 As a read-only repository, we will not provide additional releases, pull requests, issues, and importantly, **security updates**.
 A project being archived is a signal that we should not consider the project as under any active attention or maintenance.
 
-##### Possible Movement
+#### Possible Movement
 
 Ideally, all archived projects remain archived indefinitely. In extraordinary circumstances, these projects can be forked or unarchived by a conscientious developer.
 The OSPO will gladly unarchive projects for parties that find something necessary to update, but we would generally prefer a new project or a fork be made -- so that we don't archive / unarchive ad nauseam.
 This would move the project into a **maintained** or **incubating** state exclusively.
 We explicitly should not unarchive what we expect to be **preserving** projects, given that they are intentionally left alone to serve production use cases or serve some other important purpose.
+
+### Exceptional Cases
+
+In rare cases there are projects that don't match the pre-defined states. Some the reasons might be:
```

Comment:
it may be worth noting that they don't match, and don't come close to matching. Up to you as the writer -- I think it's worth noting that being close can be close enough for many states we describe. At least that's my opinion. :)